x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/alert_list_tabs.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useCallback } from 'react';
+import { EuiTabs, EuiTab } from '@elastic/eui';
+import { useHistory } from 'react-router-dom';
+import { EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { urlFromQueryParams } from './url_from_query_params';
+import { useAlertListSelector } from './hooks/use_alerts_selector';
+import * as selectors from '../../store/alerts/selectors';
+
+export const AlertListTabs = memo(() => {
+  const history = useHistory();
+  const queryParams = useAlertListSelector(selectors.uiQueryParams);
+  const isOnClosedAlertsTab = useAlertListSelector(selectors.isOnClosedAlertsTab);
+
+  const onOpenTabClick = useCallback(() => {
+    const { closed_alerts, ...paramsWithoutClosedAlerts } = queryParams;
+    history.push(urlFromQueryParams(paramsWithoutClosedAlerts));
+  }, [history, queryParams]);
+
+  const onClosedTabClick = useCallback(() => {
+    history.push(urlFromQueryParams({ ...queryParams, closed_alerts: 'true' }));
+  }, [history, queryParams]);
+
+  return (
+    <>
+      <EuiTabs size="s">
```

Comment:
I was just catching up on Pedro's weekly update and saw the GIF for this PR - Nice! 🚀 🔥 
FYI - After having worked on Ingest, I thought about adding this (tabs) functionality to our `PageView` common component as an optional prop (ex. `tabs?: EuiTabsProps`) and have that component add the tabs into the header section. I'm thinking at some point, other areas of the Endpoint app will also need it. Just a thought. Does not need to be part of this PR.